src/main/java/com/github/princesslana/eriscasper/action/VoidRouteAction.java

```diff
@@ -0,0 +1,23 @@
+package com.github.princesslana.eriscasper.action;
+
+import com.github.princesslana.eriscasper.rest.Route;
+import io.reactivex.Completable;
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface VoidRouteAction<O> extends Action {
```

Comment:
How much do we gain with a separate VoidRouteAction, over just using RouteAction<Void, T>?